jina/drivers/helper.py

```diff
@@ -152,3 +152,30 @@ def guess_mime(uri):
         tmp = urllib.request.urlopen(page)
         m_type = tmp.info().get_content_type()
     return m_type
+
+
+class DocGroundTruthPair:
```

Comment:
i see, pretty smart. but how about other attributes, `.embedding`, `.text`, etc?

Reply:
As I see, this is helpful to accompany the pair during the traversal trip, (Then we can leave the responsability to the specific drivers to interpret the specific parts they want to extract?)